packages/terra-tabs/src/orion-fusion-theme/Tab.module.scss

```diff
@@ -1,7 +1,10 @@
 @import '~terra-mixins/lib/Mixins';
 
-:local {
+:local {  
   .orion-fusion-theme {
+    --terra-tabs-add-button-focus-box-shadow: none;
```

Comment:
this variable we can remove

Reply:
Done https://github.com/cerner/terra-framework/commit/fcb1925802030dfb5be6bf821a0a978a5c82d40c